### PR TITLE
guidelines for creating a successful story narrative and technical resources 

### PR DESCRIPTION
**Pull request by CrystalOnScript**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/start/create_successful_stories.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Not a developer? Use one of many [creation tools](https://amp.dev/documentation/guides-and-tutorials/start/visual_story/?format=stories) to build your Story!
+
+
+## Story discoverability 
+
+After you’ve created your story, make sure it can be shared and discovered! All stories must meet the following requirements: 
+
+
+*   Stories must be valid AMP. You can test yours in the [AMP validator](https://validator.ampproject.org/). 
+*   [Metadata attributes](https://amp.dev/documentation/components/amp-story/#metadata-guidelines) ensure your story is working as intended. Check that your story has them setup correctly.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/start/create_successful_stories.md]: what does `ensure your story is working as intended` mean in this context?

**Comment by sebastianbenz** [comment]
@CrystalOnScript the link is broken

```
[10:51:15 PM] [Reference checker] › ☐  pending   - /content/amp-dev/documentation/components/amp-story/
819[10:51:15 PM] [Reference checker] › ℹ  info      
820[22:51:15] 'lintGrow' errored after 16 s
821[22:51:15] Error: 1 broken links and 0 wrong anchors found
822    at Pumpify.<anonymous> (/home/travis/build/ampproject/amp.dev/platform/lib/tools/growReferenceChecker.js:180:13)
823    at runMicrotasks (<anonymous>)
824    at processTicksAndRejections (internal/process/task_queues.js:94:5)
825[22:51:15] 'lintAll' errored after 16 s
```

**Comment by matthiasrohmer** [comment]
@CrystalOnScript, it needs to be `/content/amp-dev/documentation/components/reference/amp-story.md`. `reference` is missing.

**Comment by CrystalOnScript** [comment]
Thanks @matthiasrohmer :)

**Pull request merged**